Source/build_rxswift.bash

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Small comment about `bash` vs `sh`:
https://rainbowhackerhorse.github.io/bash-Is-Not-sh/
https://mywiki.wooledge.org/Bashism

Given `bash` is on macOS and that's what this script is for, I probably should not complain though :smile:.

Reply:
Yeah, that's a valid concern. I honestly don't have a strong opinion, I went for the fastest. If necessary at some point we could definitely adapt the scripts. Anyway they don't do much.